compiler/fm-equalize/fmelib/ComputeParam.py

```diff
@@ -37,3 +37,13 @@ def _channelwiseMinMax(tensors: Tensors, channel: int) -> Tuple[List[float], Lis
         channel_wise_min.append(float(np.min(min_act[:, :, :, c])))
         channel_wise_max.append(float(np.max(max_act[:, :, :, c])))
     return channel_wise_min, channel_wise_max
+
+
+def getActivationMax(tensor: np.ndarray) -> float:
+    """
+    Get max values of activation.
+    :param tensors: a list of numpy array (each is a tensor)
+    :return: a list of scale (float) for each channel
```

Comment:
Maybe?
```suggestion
    :return: a list of max (float) of absolute values for each channel
```

Reply:
That sounds more better. I just followed "activation scale" in the smooth quant [codes](https://github.com/mit-han-lab/smoothquant/blob/c61476d728e42ae0d8a35e7e78494edcac3237b5/smoothquant/smooth.py#L19).